src/Operations.jl

```diff
@@ -423,14 +423,13 @@ function install_git(
         GitTools.clone(urls[1], repo_path; isbare=true, header = "[$uuid] $name from $(urls[1])", credentials=creds)
     end
     git_hash = LibGit2.GitHash(hash.bytes)
-    for i = 2:length(urls)
+    for url in urls
```

Comment:
Maybe skip the first if you just cloned it?

Reply:
I'm not sure you get all the remotes that we give in `refspecs` when you clone?